src/GenerateOfflineMap.cpp

```diff
@@ -742,13 +748,34 @@ try {
         AnnounceStartBlock("Applying offline map to data");
 
         mapRemap.SetFillValueOverride(static_cast<float>(dFillValueOverride));
+        
+        //DAVE ADDS NEXT TWO CONDITIONALS
+        if((eInputType  == DiscretizationType_FV) &&
+        (eOutputType != DiscretizationType_FV)){
+			dataGLLNodesOut = dataGLLNodes;
+		}
+		
+		else if((eInputType  != DiscretizationType_FV) &&
+        (eOutputType == DiscretizationType_FV)){
```

Comment:
What about SE-SE ?!

Reply:
For SE-SE, there are no conditionals needed; dataGLLNodesIn, and dataGLLNodesOut are already specified in lines 629 and 631.